media/js/base/core-datalayer-init.js

```diff
@@ -4,14 +4,31 @@
 
 // init core dataLayer object and push into dataLayer
 $(function() {
-    if (Mozilla && Mozilla.Analytics) {
-        var dataLayer = window.dataLayer = window.dataLayer || [];
+    var analytics = Mozilla.Analytics;
+    var client = Mozilla.Client;
+    var dataLayer = window.dataLayer;
 
-        dataLayer.push({
+    function sendCoreDataLayer() {
+        var dataLayerCore = {
             'event': 'core-datalayer-loaded',
-            'pageId': Mozilla.Analytics.getPageId()
+            'pageId': analytics.getPageId(),
+            'pageHasDownload': analytics.pageHasDownload(),
+            'pageHasVideo': analytics.pageHasVideo(),
+            'pageVersion': analytics.getPageVersion(),
+            'testPilotUser': 'testpilotAddon' in navigator ? 'true' : 'false'
```

Comment:
I think it would be useful to add a comment here that this property is white listed for www.mozilla.org only, and we can expect it to always return false for other domains (to avoid confusion in the future).